roles/openshift_logging/tasks/install_elasticsearch.yaml

```diff
@@ -134,6 +136,8 @@
     openshift_logging_es_recover_after_time: "{{openshift_logging_es_ops_recover_after_time}}"
```

Comment:
@portante I think it is correct.

There is `{{openshift_logging_es_recover_after_time}}` value set to `${RECOVER_AFTER_TIME}` env variable in [`roles/openshift_logging/templates/es.j2`](
https://github.com/openshift/openshift-ansible/blob/39d078c8b1a17f3e4b827f9962a9f317a6464f3e/roles/openshift_logging/templates/es.j2#L87-L88) which is then used in [`elasticsearch.yml.j2` template](https://github.com/openshift/openshift-ansible/blob/39d078c8b1a17f3e4b827f9962a9f317a6464f3e/roles/openshift_logging/templates/elasticsearch.yml.j2#L37).

_//cc @ewolinetz (Erik see below please)_

However, I do not understand why this value is set only for **DeploymentConfig for Ops** in `tasks/install_elasticsearch.yaml` script. Please compare `vars` sections [for Ops](https://github.com/openshift/openshift-ansible/blob/52726611f182c6db390d7a964f0bf24fed297761/roles/openshift_logging/tasks/install_elasticsearch.yaml#L121-L136) vs `vars` section [for Non-Ops](tasks/install_elasticsearch.yaml).

Why the following four `vars` are set only for Ops DeploymentConfig?

- `es_node_quorum`
- `es_recover_after_nodes`
- `es_recover_expected_nodes`
- `openshift_logging_es_recover_after_time`

Apart from this if there is any issue with `openshift_logging_es_recover_after_time` usage and configuration then I suggest to address it in separated ticket/PR.